polyfill/lib/ecmascript.mjs

```diff
@@ -1570,6 +1573,30 @@ export const ES = ObjectAssign({}, ES2019, {
     );
   },
 
+  CreateTemporalDate: (isoYear, isoMonth, isoDay, calendar, object) => {
+    ES.RejectDate(isoYear, isoMonth, isoDay);
+    ES.RejectDateRange(isoYear, isoMonth, isoDay);
+    if (!calendar || typeof calendar !== 'object') throw new RangeError('invalid calendar');
```

Comment:
```suggestion
```

(We don't have any "assertion" capability in this file, do we?)

Reply:
Done.